### PR TITLE
ci changes to run go-licenses before check-in

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

**Review by melinath — CHANGES_REQUESTED** [review]
It would be nice to run this as a separate cloudbuild trigger from cloudbuild-tests-unit - that way, they can fail independently and be clearly displayed in the github UI

**Comment by ScottSuarez** [comment]
@melinath  presumably I'll have to merge this before I can set up such a task?

File: .ci/cloudbuild-tests-go-licenses.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# See cloud-foundation-cicd project
+
+timeout: 1800s
+steps:
+- id: test-go-licenses
+  name: 'gcr.io/graphite-docker-images/terraform-validator-builder'
+  entrypoint: /bin/bash
+  args: ['-c', 'make test-go-licenses']
+  waitFor: ['build']
```
Comment (melinath) [inline .ci/cloudbuild-tests-go-licenses.yaml:23]: I think you will need to remove this line

```suggestion
```

File: Makefile [hunk 2]
```diff
@@ -13,6 +13,11 @@ test-integration:
 	terraform --version
 	go test -v -run=CLI ./test
 
+test-go-licenses:
+	cd .. && go version && go get github.com/google/go-licenses
```
Comment (melinath) [inline Makefile:17]: Does using a `cd ..` here mean that the following line will run in the parent directory or will the working directory reset?

**Review by melinath — APPROVED** [review]
You could set up the trigger first - it's just that all in-flight PRs will fail to run the check because the file doesn't exist yet. But that will happen regardless of the order. As long as you confirm that the cloudbuild trigger works right I think it should be fine.

A couple small things but otherwise LGTM.

File: Makefile (hunk 2 above)
Comment (ScottSuarez) [reply Makefile:17]: Each line within make is sandboxed. So it won't affect the subsequent line.

https://stackoverflow.com/questions/1789594/how-do-i-write-the-cd-command-in-a-makefile

**Review by melinath — APPROVED** [review]

**Pull request merged**